recipes-qt/qt5/qtvirtualkeyboard_git.bb

```diff
@@ -17,6 +17,8 @@ PACKAGECONFIG ?= "lang-all lipi-toolkit qtquickcompiler"
 PACKAGECONFIG[hunspell] = ",CONFIG+=disable-hunspell,hunspell"
 PACKAGECONFIG[t9write] = "CONFIG+=t9write"
 PACKAGECONFIG[lipi-toolkit] = "CONFIG+=lipi-toolkit"
+PACKAGECONFIG[arrow-key-navigation] = "CONFIG+=arrow-key-navigation"
+PACKAGECONFIG[handwriting] = "CONFIG+=handwriting"
```

Comment:
handwriting isn't really useful config to use as it will just end up using lipi-toolkit, unless you have t9write sources available.